lib/propagate-steps.unit.spec.js

```
@@ -110,3 +108,29 @@ test('When propagating a repeatable instance without a namespace', function (t)
   }
   t.end()
 })
+
+test('When a single step is missing', t => {
+  const badData = getTestData('input-single-missing-step')
+
+  try {
+    propagate(badData)
```

Same as above re: possibility of silently passing